tox.ini

```diff
@@ -1,12 +1,21 @@
 [tox]
-envlist = tests, flake8
+envlist = tests, marionette, flake8
```

Comment:
If we make the environment list:

```ini
evlist = py{27,35}, py{27,35}-marionette, flake8
```

Then you'll have the following environments: py27, py35, py27-marionette, py35-marionette. This will allow you to only have py35-marionette as an allowed failure.